openfisca_france_data/common.py

```diff
@@ -395,12 +401,18 @@ def create_traitement_indiciaire_brut(individus, period = None, revenu_type = 'i
     individus['primes_fonction_publique'] = TAUX_DE_PRIME * traitement_indiciaire_brut
 
 
-def create_revenus_remplacement_bruts(individus, period, tax_benefit_system):
+def create_revenus_remplacement_bruts(individus, period, tax_benefit_system, revenu_type = 'imposable'):
     assert 'taux_csg_remplacement' in individus
 
     individus.chomage_imposable.fillna(0, inplace = True)
     individus.retraite_imposable.fillna(0, inplace = True)
-    individus.salaire_net.fillna(0, inplace = True)
+    if revenu_type == 'imposable':
+        assert 'salaire_imposable' in individus.columns
+        salaire_pour_inversion = individus.salaire_imposable
+    else:
```

Comment:
à la place du `else` je mettrai un `elif revenu_type == 'net` puis un `else: raise error not implemented`